VTEX - Logistics API.json

```diff
@@ -4326,7 +4326,7 @@
                     "Inventory"
                 ],
                 "summary": "Update inventory by SKU and warehouse",
-                "description": "Updates the [inventory](https://help.vtex.com/tutorial/inventory-management--tutorials_139) of a SKU unit in a given [warehouse](https://help.vtex.com/en/tutorial/warehouse--6oIxvsVDTtGpO7y6zwhGpb). This endpoint allows you to do the following:\r\n\r\n- **Define a quantity:** You will indicate a specific number of SKU units, and it will decrease whenever a unit is sold. The store can eventually run out of inventory.\r\n\r\n- **Set an infinite inventory:** You will always have the SKU from the informed warehouse available for sales. The store cannot run out of inventory.\r\n\r\nSuccessful requests receive a `200 OK` with `true` in the response body. If an error occurs, you will not get a `false` string in the response body, but a message instead. For example, sending a malformatted request body will result in a  `400 Bad Request` message as a response.",
+                "description": "Updates the [inventory](https://help.vtex.com/tutorial/inventory-management--tutorials_139) of a SKU unit in a given [warehouse](https://help.vtex.com/en/tutorial/warehouse--6oIxvsVDTtGpO7y6zwhGpb). This endpoint allows you to do the following:\r\n\r\n- **Define a quantity:** You will indicate a specific number of SKU units, and it will decrease whenever a unit is sold. The store can eventually run out of inventory.\r\n\r\n- **Configure lead time:** You will set the [lead time](https://help.vtex.com/en/tutorial/lead_time-shipping-time-at-sku-level--16yv5Mkj6bTyWR1hCN2f4B) for a given SKU from a warehouse, and this period will be added to the order total shipping time. This is an optional configuration.\r\n\r\n- **Set an infinite inventory:** You will always have the SKU from the informed warehouse available for sales. The store cannot run out of inventory.\r\n\r\nSuccessful requests receive a `200 OK` with `true` in the response body. If an error occurs, you will not get a `false` string in the response body, but a message instead. For example, sending a malformatted request body will result in a  `400 Bad Request` message as a response.",
```

Comment:
```suggestion
                "description": "Updates the [inventory](https://help.vtex.com/tutorial/inventory-management--tutorials_139) of a SKU in a given [warehouse](https://help.vtex.com/en/tutorial/warehouse--6oIxvsVDTtGpO7y6zwhGpb). This endpoint allows you to do the following:\r\n\r\n- **Define a quantity:** Indicate a specific number of SKUs, and it will decrease whenever a unit is sold. The store can eventually run out of inventory.\r\n\r\n- **Configure lead time:** Set the [lead time](https://help.vtex.com/en/tutorial/lead_time-shipping-time-at-sku-level--16yv5Mkj6bTyWR1hCN2f4B) for a given SKU from a warehouse, and this period will be added to the order total shipping time. This is an optional configuration.\r\n\r\n- **Set an infinite inventory:** Always have the SKU from the informed warehouse available for sales. The store cannot run out of inventory.\r\n\r\nSuccessful requests receive a `200 OK` with `true` in the response body. If an error occurs, you will not get a `false` string in the response body, but a message instead. For example, sending a malformatted request body will result in a  `400 Bad Request` message as a response.",
```